projects/chisel.yml

```
@@ -40,10 +40,10 @@ accepted_lf_code_of_conduct: not yet
 # Did the project adopt the CHIPS Alliance IP Policy? See https://technical-charter.chipsalliance.org
 adopted_chips_alliance_ip_policy: not yet
 # Did the project put links to CHIPS in the header or footer of its website? If no website, still answer "yes" as declaration of will to do so.
-chips_header_footer_text_on_website: yes
+chips_header_footer_text_on_website: not yet
```

```suggestion
chips_header_footer_text_on_website: yes
```
Done via the both the logo (which is clickable link) and in the text: https://www.chisel-lang.org